spot-ml/src/main/scala/org/apache/spot/netflow/FlowSuspiciousConnectsAnalysis.scala

```diff
@@ -61,39 +68,18 @@ object FlowSuspiciousConnectsAnalysis {
     val invalidFlowRecords = filterAndSelectInvalidFlowRecords(inputFlowRecords)
     dataValidation.showAndSaveInvalidRecords(invalidFlowRecords, config.hdfsScoredConnect, logger)
 
+    outputFlowRecords
```

Comment:
it's an artifact of something I was doing during testing ; I will remove it and leave these changes to SPOT-128